src/osd/OpRequest.h

```diff
@@ -64,6 +64,7 @@ struct OpRequest : public TrackedOp {
   uint8_t hit_flag_points;
   uint8_t latest_flag_point;
   utime_t dequeued_time;
+  std::string delayed_detail;
```

Comment:
Using a string here is going to cause extra allocations and copies in the fast path.  A quick grep suggests that all callers of mark_delayed actually pass const char* literals.  How about we instead store a const char*?  Can we eliminate mark_flag_point_string() entirely?

I also don't think it's a good idea to modify state_string() -- it should simply be a single string for each flag.  How about we leave that alone but add a detail const char* that any flagpoint can set?

Reply:
You are right, let me think again.

Reply:
Not if the pointer is to a string constant in the data segment.

Reply:
I got it